tools/theme-preview.sh

```diff
@@ -2,31 +2,84 @@
 #shellcheck disable=SC2317
 set -u
 
-COLUMNS=160
-
-. ./liquidprompt --no-activate
-
-# If a theme was given, use the theme and source all other arguments.
-if [[ -z ${1-} || $1 == --help ]]; then
+usage() {
   printf '
-Usage: %s theme [sourced files...]
+Usage: %s theme [options...] [sourced files...]
 
 Print out example prompts based on a standard set of input conditions. Designed
 to showcase Liquid Prompt themes.
 
-Example usage: %s powerline themes/powerline/powerline.theme
+Options:
+  --config-file=<filename> Load <filename> as an additional Liquid Prompt
+                           config file.
+  --user-config            Load the user, or if not found, the system config
+                           file.
+  --reproducible           Set the terminal size to a static value. Load no
+                           config files. Overrides any --config-file or
+                           --user-config arguments.
+  -h, --help               Print this help text.
```

Comment:
I would sort the options in the reverse order, which is going from the simplest to the more complex: first the _help_, then _reproducible_, then _user-config_, then _config-file_.

Reply:
And maybe indicate here that presets are config files?